### ISSUE TITLE
Equation number overlaps equation

### ISSUE THREAD

**Issue by Kaba2** [mathjax/MathJax#1189]
I am having problems with the automatic equation numbering (autoNumber: "all") overlapping a long equation:

![number_overlap](https://cloud.githubusercontent.com/assets/2399754/7556298/dcd65b22-f773-11e4-92a5-4a2debbf4784.png)

The html for this is given by

```html
<div class="Example">
<p>A function <span class="latex-math"><script type="math/tex">f : \mathbb{R}^n \to \mathbb{R}^m</script></span> is <em>continuous</em> at <span class="latex-math"><script type="math/tex">p</script></span>, if</p>
<p><span class="display-latex-math"><script type="math/tex; mode=display">\forall \varepsilon \in \mathbb{R}^{> 0}: \exists \delta \in \mathbb{R}^{> 0}: \forall x \in \mathbb{R}^{d} : \left\lVert x - p \right\rVert < \delta \implies \left\lVert f(x) - f(p) \right\rVert < \varepsilon.</script></span></p>
</div>
```

and the CSS is given by

```css
.Example
{
    padding: 0.75em;
    margin-top: 1em;
    margin-bottom: 1em;
    border: 0.25em solid #8A8;
    overflow: auto;
}
```

Any ideas?

**Comment by pkra** [comment]
We'd need a live sample to rule out a bug.  But the screenshot indicates that the container is simply not wide enough to fit both the equation and it's label. So something has to give. (Automatic) line-breaking would help. There are currently no options for controlling this situation more generally.

Related are #957, #1190, #596 and others. Marking this duplicate for now.

**Comment by Kaba2** [comment]
Here is a minimal sample. These files should be stored in the same directory.

test.html
---

```html
<!DOCTYPE html>
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1">
<title></title>
<link rel="stylesheet" type="text/css" href="test.css"/>
<script type="text/javascript" src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?delayStartupUntilConfig"></script>
<script type="text/javascript" src="mathjax-config.js"></script>
</head>
<body>
<div id = "remark-all">
<div id = "remark">
<div class="Example">
<p><span class="display-latex-math"><script type="math/tex; mode=display">\forall \varepsilon \in \mathbb{R}^{> 0}: \exists \delta \in \mathbb{R}^{> 0}: \forall x \in \mathbb{R}^n : \left\lVert x - p \right\rVert < \delta \implies \left\lVert f(x) - f(p) \right\rVert < \varepsilon.</script></span></p>
</div>
</div>
</div>
</body>
</html>
```

test.css
---

```css
html, body 
{
    margin: 0em;
    padding: 0em;
}

#remark-all
{
    font-family: Sans-serif;
    max-width: 40em;
    margin: auto;
    padding: 0.5em;
}

#remark
{
    line-height: 1.3em;
    min-height: 100vh;
}

#remark .Example
{
    padding: 0.75em;
    margin-top: 1em;
    margin-bottom: 1em;
    border: 0.25em solid #8A8;
    overflow: auto;
}
```

mathjax_config.js
---

```javascript
MathJax.Hub.Config({
    config: 
    [
        "MMLorHTML.js"
    ],
    jax: 
    [
        "input/TeX",
        //"input/MathML",
        "input/AsciiMath",
        "output/HTML-CSS",
        "output/NativeMML", 
        "output/CommonHTML"
    ],
    extensions: 
    [
        "tex2jax.js",
        //"mml2jax.js",
        "asciimath2jax.js",
        "MathMenu.js",
        "MathZoom.js", 
        "CHTML-preview.js"
    ],
    TeX: 
    {
        extensions: 
        [
            "AMSmath.js",
            "AMSsymbols.js",
            "noErrors.js",
            "noUndefined.js"
        ],
        equationNumbers: 
        { 
            autoNumber: "all" 
        }    
    },
    asciimath2jax: 
    {
        delimiters: [],
        ignoreClass: 'remark-all',
        processClass: 'ascii-math'
    },
    tex2jax: 
    {
        inlineMath: [],
        displayMath: [],
        ignoreClass: 'remark-all',
        processClass: 'latex-math|display-latex-math'
    }
});

MathJax.Hub.Configured();
```

**Comment by dpvc** [comment]
The issue was *supposed* to have been fixed in v2.5 (in both SVG and HTML-CSS output), but it turns out that an incorrect scaling factor was applied to the width, and the wrong minimum width is being used.  

What's supposed to happen is that the equation and its number will extend beyond the continuing box (or perhaps force it to a larger size depending on the situation), but that may not be what you really want, either.  As Peter says, something has to give in this situation.

**Comment by Kaba2** [comment]
The box has overflow: auto, so I would expect the equation to be as long as necessary, and the scroll-bar to appear?

**Comment by dpvc** [comment]
Yes, in your case, that is shat should happen (and does when the incorrect scaling factor is fixed).  I had actually diagnosed the problem before you sent in the example, and hadn't looked closely at it when I wrote my previous message.  Sorry for the confusion.

**Comment by Kaba2** [comment]
Ok, thanks for the explanation. I'll just wait for a fix then:)

**Comment by dpvc** [comment]
For the time being, you could add `\hskip 2.5em` to the beginning and ending of your formula to make it wider so the number would overlap the space rather than the equation itself.  A temporary fix if you are willing to go back and take that out when the problem is fixed in MathJax

**Comment by Kaba2** [comment]
Thanks. What might be the expected publication schedule for version 2.6?

**Comment by pkra** [comment]
Please note that this issue has not been added to the 2.6 milestone (though it seems likely that we will add it). Also, there is no ETA for v2.6 yet.

**Comment by pkra** [comment]
@dpvc this was marked "duplicate" but reading through the issue I'm not so sure anymore. Could you relabel or close?

**Comment by dpvc** [comment]
I believe it is a duplicate of #596, and that the fix for that will take care of this problem.  The handling of spacing for labels (and indentation and the other related factors) was reworked for the 596 solution.  I think that all the issues dealing with overlapping labels should be resolved at this point.
